app/controllers/api/v1/graphql_controller.rb

```diff
@@ -35,7 +35,7 @@ def batch
               id: result.query.context[:id],
               payload: result.to_h
             }
-          end
+          end unless queries.nil?
```

Comment:
I didn't understand this `unless`... aren't we defining `queries` for the first time in this block?

Reply:
`queries` is defined but if `params[:_json]` is nil then `queries` will be nil and expected that `RelayOnRailsSchema.multiplex(queries)` with raise an error so I added this condition

Reply:
Ah OK, the Git diff confused me... so basically we're handling the case where the batch endpoint is called but no queries are passed to it?

Reply:
yes